README.md

```diff
@@ -492,6 +494,41 @@ status:
     status: 'True'
 ```
 
+## Direct Secret Reference
+
+There are scenarios where Provisioned Service is not available.  But there is a `Secret` resource available for binding.  This extension, named Direct Secret Reference, allows Service Binding resource with the service directly pointing to a secret resource.
+
+The `Secret` **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  It is **RECOMMENDED** that the `Secret` also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` **MAY** contain any other entry.
```

Comment:
We should avoid duplicating requirements outlined elsewhere in the spec unless we are amending them.
```suggestion
```